Project.toml

```
@@ -51,6 +52,7 @@ Markdown = "1.6"
 OffsetArrays = "1.1.3"
 Optim = "1.7.2"
 OrderedCollections = "1.3.0"
+PDMats = "0.11.35"
```

It'll be easier to check when the test suite is running (the downgrade jobs will check this), but I think we can relax these bounds to those of our Distributions lower bound:
```suggestion
PDMats = "0.10, 0.11"
```